client/webserver/webserver.go

```diff
@@ -625,6 +627,34 @@ func New(cfg *Config) (*WebServer, error) {
 	return s, nil
 }
 
+// fetchLatestVersion is a helper function to retrieve the latest version of the app
+// from github.
+func (w *WebServer) fetchLatestVersion(ctx context.Context) {
+	ticker := time.NewTicker(1 * time.Minute)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
```

Comment:
I may still be missing something, but this could fire once without waiting, as soon as the app is started, then go into the loop if that fails.